examples/applications/author-disambiguation/ethnicity.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Beard.
+# Copyright (C) 2015 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Author disambiguation -- Build an estimator for guessing an author ethnic
+group from his name.
+
+See README.rst for further details.
+
+.. codeauthor:: Gilles Louppe <g.louppe@cern.ch>
+.. codeauthor:: Hussein Al-Natsheh <hussein.al.natsheh@cern.ch>
+
+"""
+
+import argparse
+import numpy as np
+import pandas as pd
+import pickle
+
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.pipeline import Pipeline
+from sklearn.svm import LinearSVC
```

Comment:
Linear classifier are often good enough for tf-idf features. I had tried a few alternatives but this one gave decent accuracy without trying too hard.